applications/app/controllers/AtomPageController.scala

```diff
@@ -0,0 +1,63 @@
+package controllers
+
+import com.gu.contentapi.client.model.v1.ItemResponse
+import common._
+import model.Cached.{RevalidatableResult, WithoutRevalidationResult}
+import model._
+import play.api.mvc._
+import play.twirl.api.Html
+import scala.concurrent.Future
+import contentapi.ContentApiClient
+import model.content._
+import views.html.fragments.atoms.{ media => MediaAtomBody, storyquestions => StoryQuestionsAtomBody }
+
+class AtomPageController(contentApiClient: ContentApiClient)(implicit context: ApplicationContext) extends Controller with Logging with ExecutionContexts {
+
+  def render(atomType: String, id: String) = Action.async { implicit request =>
+
+    lookup(s"atom/$atomType/$id") map {
+      case Left(model: MediaAtom) =>
+        (MediaAtomPage(model), MediaAtomBody(model, displayCaption = false, mediaWrapper = Some(MediaWrapper.EmbedPage)))
+      case Left(model: StoryQuestionsAtom) =>
+        (StoryQuestionsAtomPage(model), StoryQuestionsAtomBody(model, isAmp = false))
+      case Left(model: Quiz) => /* noop */
```

Comment:
ha! in fact I must do that because Atom is an ADT and the compiler will complain. these cases will never happen until we specifically intent to, so I think it's fine.